runtime/compiler/aarch64/runtime/PicBuilder.spp

```diff
@@ -85,6 +91,14 @@
 	.set	J9TR_USCSnippet_HelperOffset,	0xFF000000
 	.set	J9TR_USCSnippet_HelperOffsetShift,	24
 
+// Interface call snippet
+
+	.set	J9TR_ICSnippet_codeCacheReturnAddress,	0
+	.set	J9TR_UICSnippet_CP,		8
+	.set	J9TR_UICSnippet_CPIndex,	16
+	.set	J9TR_ICSnippet_InterfaceClass,	20
```

Comment:
If we change the size of cpIndex in the snippet then these offsets will need to be adjusted.

Reply:
Adjusted the offsets.